README.md

```diff
@@ -29,18 +37,25 @@ Or go to _Settings → Install_ and search for `prettier-atom`.
 
 Make sure to restart Atom after the package is installed.
 
-## Usage
+## Detailed Usage
```

Comment:
This is good. Looking at the content below, I think we should split the content into some nested h3 sections?:

- How to Format
  - Format on Save
  - Manual Formatting
- What Version of Prettier Gets Used?
- ESLint Integration
- Editorconfig Integration
- Stylelint Integration